### ISSUE TITLE
2.0-SNAPSHOT: springfox.documentation.swagger.configuration.SpringSwaggerConfig missing / README.md needs update

### ISSUE THREAD

**Issue by jfiala** [springfox/springfox#656]
I just wanted to try the latest version (2.0.0-SNAPSHOT):

The class springfox.documentation.swagger.configuration.SpringSwaggerConfig
seems now completely missing in the sources (2.0.0-SNAPSHOT).

Additionally, SwaggerSpringMvcPlugin has also vanished.

Can you please update README.md which files should be now used instead?

Thank you,
J. Fiala

**Comment by dilipkrish** [comment]
@jfiala have you looked at the [preliminary documentation](https://github.com/springfox/springfox/blob/master/transitioning-to-v2.md) to transition to 2.0.0?

**Comment by jfiala** [comment]
thank you, I thought the main documentation page is up to date. I'll have a look at the preliminary documentation.

**Comment by dilipkrish** [comment]
@jfiala the main documentation page *IS* [uptodate](https://github.com/springfox/springfox#usage-guide-and-instructions-to-migrate-to-20x).

**Comment by jfiala** [comment]
OK, but maybe it would make sense to add the different dependency for 1.0.2 in the "Usage (Quick guide) for 1.0.x"?

``` xml
<dependency>
		    <groupId>com.mangofactory</groupId>
		    <artifactId>swagger-springmvc</artifactId>
		    <version>1.0.2</version>
		</dependency>
```

Or simply remove the whole 1.x section in the main documentation because there is already a redirect at the top ("NOTE: Version 2.x has not yet been released. To use 1.0.2 released version please browse to the pre-2.0 tag")

**Comment by dilipkrish** [comment]
Closing this as a duplicate for #531
